monai/transforms/post/array.py

```diff
@@ -361,22 +361,68 @@ class RemoveSmallObjects(Transform):
     Data should be one-hotted.
 
     Args:
-        min_size: objects smaller than this size (in pixel) are removed.
+        min_size: objects smaller than this size (in number of voxels; or volume in mm^3 if in_mm3 is True) are removed.
         connectivity: Maximum number of orthogonal hops to consider a pixel/voxel as a neighbor.
             Accepted values are ranging from  1 to input.ndim. If ``None``, a full
             connectivity of ``input.ndim`` is used. For more details refer to linked scikit-image
             documentation.
         independent_channels: Whether or not to consider channels as independent. If true, then
             conjoining islands from different labels will be removed if they are below the threshold.
             If false, the overall size islands made from all non-background voxels will be used.
+        in_mm3: Whether the specified min_size is in number of voxels or volume in mm^3, default is false.
```

Comment:
So I would call this instead `by_measure` and say if this is True then `min_size` is a surface area or volume value of whatever units your image is in (mm^3, cm^2, etc.).